app/decorators/reviewable.rb

```diff
@@ -21,4 +30,7 @@ def reviewed?
     !!workflow_metadata.reviewed
   end
 
+  def needs_reviewed
```

Comment:
This returns true if it's already been reviewed - pretty sure that's backward, given the method name